### ISSUE TITLE
Gsoc 2021 introduction 

### ISSUE THREAD

**Issue by sparkingdark** [salesforce/gsoc#18]
Hi @alyssaa  I am Debojyoti want to contribute and participate in this org.How to start?

**Comment by alyssaa** [comment]
Hi Debojyoti,

Thanks for reaching out! Organizations will be announced on March 9 through
Google Summer of Code. If you are interested in participating through
Google Summer of Code you can apply through their intern program.

Thank you!

*Alyssa Arvin *

*Senior Program Manager, Open Source*

 415.471.4603 | aarvin@salesforce.com






<https://smart.salesforce.com/sig/aarvin//us_mb/default/link.html>

**Comment by sparkingdark** [comment]
Okay thanks for clarifying.but it's a mentorship program

**Comment by alyssaa** [comment]
Yes, Google Summer of Code is a mentorship program. If you are interested
in contributing to the project and not be a part of GSoC you can definitely
do that as well. Each project has open issues on their GitHub pages. Hope
that helps!

*Alyssa Arvin *

*Senior Program Manager, Open Source*

 415.471.4603 | aarvin@salesforce.com






<https://smart.salesforce.com/sig/aarvin//us_mb/default/link.html>
